renovate.json

```diff
@@ -24,6 +24,9 @@
     }, {
       "matchPackageNames": ["juju"],
       "allowedVersions": "<3.0.0"
+    }, {
+      "matchPackageNames": ["packaging"],
+      "allowedVersions": "<23.0.1"
```

Comment:
I don't think we need this if we are pinning in `pyproject.toml`

Reply:
I'm going to remove it. Thanks for catching it.